Common/Systems/BossRushInjectionSystem.cs

```diff
@@ -23,7 +23,7 @@ namespace TheBereftSouls.Common.Systems;
 //
 // Referenced permalink:
 // - https://github.com/CalamityTeam/CalamityModPublic/blob/a5bdc9231f2859abddb85c9043413a57fcb042b9/ModSupport/ModCalls.cs#L1999
-using BossRushEntry = (int, int, Action<int>, int, bool, float, int[], int[]);
+using BossRushEntry = (int, int, Action<int>, int, bool, float, int[], int[]?);
```

Comment:
Why is this last entry nullable? In the `AddToBossRush` function we do default it to `[NPCType]`.

Reply:
written in the pr description. the calamity source code may put null in the last entry, which seems to be a default to `[NPCType]` (handled inside calamity).